include/sched.h

```diff
@@ -252,9 +252,10 @@ int    sched_setaffinity(pid_t pid, size_t cpusetsize,
                          FAR const cpu_set_t *mask);
 int    sched_getaffinity(pid_t pid, size_t cpusetsize, FAR cpu_set_t *mask);
 int    sched_cpucount(FAR const cpu_set_t *set);
-int    sched_getcpu(void);
 #endif /* CONFIG_SMP */
 
+int    sched_getcpu(void);
```

Comment:
Why do we need to have this as a call in non-SMP case? I mean why we can't define `sched_getcpu` to zero in non-SMP case?

Reply:
but it's required to change many place

Reply:
Could you please expand your answer more? I'm expecting only adding `#else` fo above `#ifdef` and define in this header file. What other changes are needed?